clipboard/ios/Plugin/Clipboard.swift

```diff
@@ -0,0 +1,67 @@
+import Foundation
+import Capacitor
+
+@objc public class Clipboard: NSObject {
+
+    @objc enum ContentType: Int {
+        case string
+        case url
+        case image
+    }
+
+    @objc func write(content: String, ofType type: ContentType) {
+        switch type {
+        case ContentType.string:
+                UIPasteboard.general.string = content
+        case ContentType.url:
+            if let url = URL(string: content) {
+                UIPasteboard.general.url = url
+            }
+        case ContentType.image:
+            if let data = Data(base64Encoded: getCleanData(content)), let image = UIImage(data: data) {
+                CAPLog.print("Loaded image", image.size.width, image.size.height)
+                UIPasteboard.general.image = image
+            } else {
+                CAPLog.print("Unable to encode image")
+            }
+        default:
+            CAPLog.print("Unknown clipboard content type")
+        }
+    }
+
+    @objc func read() -> [String: Any] {
+        if let stringValue = UIPasteboard.general.string {
+            return [
+                "value": stringValue,
+                "type": "text/plain"
+            ]
+        }
+
+        if let url = UIPasteboard.general.url {
+            return [
+                "value": url.absoluteString,
+                "type": "text/plain"
+            ]
+        }
+
+        if let image = UIPasteboard.general.image {
+            let data = image.pngData()
+            if let base64 = data?.base64EncodedString() {
+                return [
+                    "value": "data:image/png;base64," + base64,
+                    "type": "image/png"
+                ]
+            }
+        }
+
+        return [:]
+    }
+
+    private func getCleanData(_ data: String) -> String {
+        let dataParts = data.split(separator: ",")
+        if let part = dataParts.last {
+            return String(part)
+        }
+        return data
+    }
```

Comment:
this should be in a Capacitor's helper class, there is a similar method on Filesystem plugin

Reply:
Discussed with Ian and Dan. We're not sure its a great idea to move this into core. It's only used in 2 plugins and its a fairly simple string manipulation function. Moving into core would require creating a new helper class just for this in iOS and adds another dependency on the core we don't think is worth it.

Reply:
I mean putting it, in example in `CAPFile.swift` where we have file related helper methods.
Don't understand the "Add another dependency on the core" part, what dependency would it add? It would be a static method plugins can call.

Reply:
Related: https://github.com/ionic-team/capacitor/pull/3457

Reply:
Dependency probably wasn't the best choice of words. I meant that it adds another coupling with the core when the code could just exist in the plugin itself and not depend on the core being there necessarily.